wpimath/src/test/python/geometry/test_transform2d.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import math
+import numpy as np
+
+from wpimath.geometry import (
+    Rotation2d,
+    Transform2d,
+    Translation2d,
+    Pose2d
+)
+from wpimath.units import meters
+
+
+def test_to_matrix():
+    before = Transform2d(x=1, y=2, rotation=Rotation2d.fromDegrees(20))
+    after = Transform2d.fromMatrix(before.toMatrix())
+    assert before == after
+
+
+def test_inverse():
+    initial = Pose2d(x=1, y=2, rotation=Rotation2d.fromDegrees(45))
+    transform = Transform2d(Translation2d(meters(5), meters(0)), Rotation2d.fromDegrees(5))
```

Comment:
Same feedback [about type aliases usage](https://github.com/wpilibsuite/allwpilib/pull/8461#discussion_r2602406226) applies to this PR.